crypto/rand/drbg_lib.c

```diff
@@ -41,73 +41,75 @@ typedef struct drbg_global_st {
     /*
      * The three shared DRBG instances
      *
-     * There are three shared DRBG instances: <master>, <public>, and <private>.
+     * There are three shared DRBG instances: <primary>, <public>, and
+     * <private>.  The <public> and <private> DRBGs are secondary ones.
```

Comment:
Here (though it is mentioned below), I'd be tempted to add: 
```
They're used for non-secret (e.g. nonces) and secret (e.g. private keys) data respectively.
```
So that it is immediately clear what "public" vs. "private" are about.  This is however a source file, and so not essential.